prusti-tests/tests/verify_overflow/fail/ghost-constraints/different-bounds.rs

```diff
@@ -0,0 +1,23 @@
+// Note: For now, it is completely fine to _declare_ two ghost constraints with different bounds
+// since resolving happens for specific callsites. That is, without the call in main, this
+// file verifies.
```

Comment:
This is not great but we're going to have multiple ghost constraints later anyway so that's fine.

Reply:
Yeah, that's indeed a bit special with the current implementation. I guess when we'll implement behavioral subtyping checks of ghost constraints, we need to preprocess all of them somehow and catch these errors there.